draft-ietf-quic-http.md

```diff
@@ -857,14 +910,17 @@ HTTP_INTERRUPTED_HEADERS (0x0E):
 : A HEADERS frame without the End Header Block flag was followed by a frame
   other than HEADERS.
 
-HTTP_SETTINGS_ON_WRONG_STREAM (0x0F):
-: A SETTINGS frame was received on a request control stream.
+HTTP_WRONG_STREAM (0x0F):
+: A frame was received on a different stream from where it is permitted.
```

Comment:
I took Ryan's text, but I moved that to #712.  This PR will appear to be include the change, but that's just because I rebased on top of #712 on the assumption that it will be merged.